pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+## Requirements
+
+- Filling out the template is required. Any pull request that does not include enough information to be efficiently reviewed may be rejected.
+- Please consider which branch this is to be submitted against. This will normally be the development branch, so please ask your reviewer if you think it needs to go somewhere else. 
+
```

Comment:
This section i.e. Requirements can go in as comments, so that it won't come up in the PR description in case the user forgot to remove it.

Reply:
That's done.